Associacao-Brasileira-de-Normas-tecnicas-ufs.csl

```diff
@@ -5,7 +5,7 @@
     <title-short>ABTN-UFS</title-short>
     <id>http://www.zotero.org/styles/Associação-Brasileira-de-Normas-tecnicas-ufs</id>
     <link href="http://www.zotero.org/styles/original-style" rel="template"/>
-    <link href="http://www.zotero.org/styles/Associação-Brasileira-de-Normas-tecnicas-ufs" rel="self"/>
+    <link href="http://www.zotero.org/styles/Associacao-Brasileira-de-Normas-tecnicas-ufs" rel="self"/>
```

Comment:
This should be all lowercase, i.e.:

`<link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufs" rel="self"/>`